sn_transfers/src/lib.rs

```diff
@@ -42,7 +42,7 @@ use lazy_static::lazy_static;
 /// The following PKs shall be updated to match its correspondent SKs before the formal release
 ///
 /// Foundation wallet public key (used to receive initial disbursment from the genesis wallet)
-const FOUNDATION_PK_STR: &str = "8f73b97377f30bed96df1c92daf9f21b4a82c862615439fab8095e68860a5d0dff9f97dba5aef503a26c065e5cb3c7ca"; // DevSkim: ignore DS173237
+const FOUNDATION_PK_STR: &str = "8f100726db5ea16d2b9660f74e4c36f853329f7544b3d268d925046f3c9c5ba33058b52cb8ff529cda9450e7f3bce6db"; // DevSkim: ignore DS173237
```

Comment:
the sk in the `audit` step within the memcheck test needs to be updated as well.
I may add in a verfication step within the memcheck step to confirm the discord name got decrypted properly.